src/schematools/permissions/db.py

```diff
@@ -9,15 +9,14 @@
 from sqlalchemy.engine import Engine
 from sqlalchemy.orm import Session, sessionmaker
 
+from schematools.permissions import PUBLIC_SCOPE
```

Comment:
Doesn't this lead to an import cycle? It did the last time I tried removing the constant from this file.

Reply:
When running the tests, this module is touched, but I don't get an import cycle. Is there a way to trigger it?

Reply:
Should be ok then.